src/Feature/Editor/EditorConfiguration.re

```diff
@@ -0,0 +1,75 @@
+open Revery;
+open Oni_Core;
+open Config.Schema;
+
+module CustomDecoders: {
+  let whitespace: Config.Schema.decoder([ | `All | `Boundary | `None]);
+  let lineNumbers: Config.Schema.decoder([ | `On | `Relative | `Off]);
+  let time: Config.Schema.decoder(Time.t);
+} = {
+  open Json.Decode;
+
+  let whitespace =
```

Comment:
Sorry, I just realized this PR I merged collides and will need to be brought over, too

https://github.com/onivim/oni2/pull/1421

I'll hold off on any other configuration changes until after this PR-

Reply:
No worries. There's not much that can conflict directly. On the contrary, this looks like a great example and test case for a pretty advanced custom configuration setting!